exercises/forth/example_gen.go

```diff
@@ -22,78 +22,54 @@ func main() {
 
 // The JSON structure we expect to be able to unmarshal into
 type js struct {
-	ParsingAndNumbers  section `json:"parsing and numbers"`
-	Addition           section
-	Subtraction        section
-	Multiplication     section
-	Division           section
-	CombinedArithmetic section `json:"combined arithmetic"`
-	Dup                section
-	Drop               section
-	Swap               section
-	Over               section
-	UserDefinedWords   section `json:"user-defined words"`
+	Exercise string
```

Comment:
I wouldn't mind if you omit these fields from the struct if they won't be used in the generator.